src/ontology/mondo-edit.obo

```diff
@@ -125013,31 +124996,41 @@ id: MONDO:0006702
 name: chronic inflammatory demyelinating polyradiculoneuropathy
 def: "A rare neurological disorder in which there is inflammation of nerve roots and peripheral nerves and destruction of the fatty protective covering (myelin sheath) over the nerves. This affects how fast the nerve signals are transmitted and leads to loss of nerve fibers. This causes weakness, paralysis and/or impairment in motor function, especially of the arms and legs (limbs). Sensory disturbance may also be present. The motor and sensory impairments usually affect both sides of the body (symmetrical), and the degree of severity and the course of disease may vary from case to case. Some affected individuals may follow a slow steady pattern of symptoms while others may have symptoms that stabilize and then relapse." [https://rarediseases.org/rare-diseases/chronic-inflammatory-demyelinating-polyneuropathy/]
 subset: gard_rare {source="GARD:6102"}
+subset: inferred_rare
 subset: orphanet_rare {source="Orphanet:2932"}
 subset: rare
+synonym: "chronic inflammatory demyelinating polyneuritis" EXACT [DOID:2536]
+synonym: "chronic inflammatory demyelinating polyneuropathy" EXACT [NCIT:C84636]
 synonym: "chronic inflammatory demyelinating polyradiculoneuropathy" EXACT [Orphanet:2932]
 synonym: "chronic relapsing polyneuropathy" EXACT [https://rarediseases.org/rare-diseases/chronic-inflammatory-demyelinating-polyneuropathy/]
 synonym: "CIDP" EXACT ABBREVIATION [https://rarediseases.org/rare-diseases/chronic-inflammatory-demyelinating-polyneuropathy/, OMIM:139393, Orphanet:2932]
+xref: DOID:2536
 xref: DOID:5213 {source="MONDO:equivalentTo", source="EFO:1000868"}
 xref: EFO:1000868 {source="MONDO:equivalentTo"}
 xref: GARD:6102 {source="Orphanet:2932"}
-xref: ICD10CM:G61.81 {source="DOID:5213", source="MONDO:directSiblingOf"}
-xref: ICD9:357.81 {source="DOID:5213", source="MONDO:directSiblingOf"}
+xref: ICD10CM:G61.81 {source="DOID:5213", source="MONDO:equivalentTo", source="DOID:2536"}
+xref: ICD9:357.81 {source="DOID:5213", source="MONDO:equivalentTo", source="MONDO:i2s", source="DOID:2536"}
 xref: ICD9:357.89 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: MedDRA:10057645 {source="EFO:1000868"}
 xref: MESH:D020277 {source="DOID:5213", source="MONDO:equivalentTo", source="EFO:1000868"}
-xref: NCIT:C84636 {source="DOID:5213", source="MONDO:directSiblingOf"}
+xref: MESH:D020277 {source="DOID:2536"}
```

Comment:
should this line `xref: MESH:D020277 {source="DOID:2536"}` have been combined with `xref: MESH:D020277 {source="DOID:5213", source="MONDO:equivalentTo", source="EFO:1000868"}`?

Reply:
yes, good catch!